run.py

```diff
@@ -0,0 +1,492 @@
+#!/usr/bin/env python3
+
+import base64
+import copy
+import csv
+import datetime
+import dateutil.parser
+import flywheel
+import json
+import logging
+import os
+import pprint
+import pytz
+import re
+import requests
+import shutil
+import sys
+import tempfile
+import zipfile
+from dicomweb_client.api import load_json_dataset
+from flywheel_migration.dcm import DicomFile
+from flywheel_migration.util import DEFAULT_TZ
+from healthcare_api.client import Client as HealthcareAPIClient
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+from urllib.parse import urljoin
```

Comment:
Minor complaint here: `urllib` is in core, `pytz` and `requests` isn't, and there's no empty line dividing core/3rd-party imports.